packages/clerk-js/src/v4/UserProfile/MfaSection.tsx

```diff
@@ -1,82 +1,49 @@
-import { PhoneNumberResource } from '@clerk/types/src';
 import React from 'react';
 
-import { useCoreUser } from '../../ui/contexts';
+import { useCoreUser, useEnvironment } from '../../ui/contexts';
 import { useNavigate } from '../../ui/hooks';
-import { Badge, Col, Icon } from '../customizables';
-import { FormattedPhoneNumberText, useCardState } from '../elements';
-import { Mobile } from '../icons';
-import { handleError } from '../utils';
-import { LinkButtonWithDescription } from './LinkButtonWithDescription';
+import { MfaPhoneCodeAccordion } from './MfaPhoneCodeAccordion';
+import { MfaTOTPAccordion } from './MfaTOTPAccordion';
 import { ProfileSection } from './Section';
-import { UserProfileAccordion } from './UserProfileAccordion';
 import { AddBlockButton } from './UserProfileBlockButtons';
-import { defaultFirst } from './utils';
+import { defaultFirst, getSecondFactors, getSecondFactorsAvailableToAdd } from './utils';
 
 export const MfaSection = () => {
   const { navigate } = useNavigate();
+  const {
+    userSettings: { attributes },
+  } = useEnvironment();
   const user = useCoreUser();
+
+  const secondFactors = getSecondFactors(attributes);
+  const secondFactorsAvailableToAdd = getSecondFactorsAvailableToAdd(attributes, user);
+
+  const showTOTP = secondFactors.includes('totp') && user.totpEnabled;
+
   const mfaPhones = user.phoneNumbers
     .filter(ph => ph.verification.status === 'verified')
     .filter(ph => ph.reservedForSecondFactor)
     .sort(defaultFirst);
 
   return (
     <ProfileSection
-      title='Two-step authentication'
+      title='Multifactor authentication'
```

Comment:
❓ Did we change the copy here again? 🙃 Some designs state multifactor, others have Two-step. Creating a slack thread so we can finalise this because I remember changing the copy here at least 2 times already!

Reply:
I got this from Figma

Agreed, we should use a single term here